index.html

```diff
@@ -64,5 +64,155 @@ <h1>
         </ul>
       </div>
     </section>
+
+    <!-- My recent Works Section -->
+
+    <section>
+      <div class="jobs" id="jobs-id">
```

Comment:
- [ ] Excellent work so far, you have done really well :clap: , I just have some suggestions to improve the design, so I think there is supposed to be a gap or spacing between the headline and works section currently it is missing that, please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/234956477-7c3976f9-d3a5-44b8-bd0f-9fd5d6ada9eb.png)
 figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/234956657-7b605077-7062-446e-b7e4-cfd63bcc5fc4.png)

I think a margin-top of 80px should fix that :+1: